packages/Webkul/Admin/src/Resources/lang/en/app.php

```diff
@@ -1677,6 +1679,9 @@
             'show-search-input-field'               => 'Show Search Input Field',
             'allow-no-of-new-product-homepage'      => 'Allowed number of New Product in Homepage',
             'allow-no-of-featured-product-homepage' => 'Allowed number of Featured Product in Homepage',
+            'allow-no-of-related-productpage'       => 'Allowed number of Related Product in Productpage',
+            'allow-no-of-up-sells-productpage'      => 'Allowed number of Up-Sells Product in Productpage',
+            'allow-no-of-cross-sells-productpage'   => 'Allowed number of Cross-Sells Product in Productpage',
```

Comment:
Earlier we are using the `kebab-case` but on improvement, we are focussing full array keys as `snake_case`. So add these as `snake_case` and the other one we will improvise in a single PR.